improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -509,6 +511,8 @@ def set_up_probability_cube(
     cube.coord(variable_name).var_name = "threshold"
     cube.coord(variable_name).attributes.update(coord_attributes)
     cube.coord(variable_name).units = Unit(threshold_units)
+    if len(thresholds) == 1:
+        cube = next(cube.slices_over(variable_name))
```

Comment:
If this was in set_up_variable_cube, you wouldn't need to do it twice!

Reply:
True - but see comment above.